core/src/main/scala/org/broadinstitute/dsde/rawls/dataaccess/HttpExecutionServiceDAO.scala

```diff
@@ -27,8 +27,8 @@ class HttpExecutionServiceDAO( executionServiceURL: String, submissionTimeout: F
     import system.dispatcher
 
     val pipeline = addAuthHeader(userInfo) ~> sendReceive ~> unmarshal[Seq[Either[ExecutionServiceStatus, ExecutionServiceFailure]]]
-    val formData = FormData(Seq("wdlSource" -> wdl, "workflowInputs" -> inputs.mkString("[", ",", "]")) ++ options.map("workflowOptions" -> _).toSeq)
-    pipeline(Post(url,formData))
+    val formData = Map("wdlSource" -> BodyPart(wdl), "workflowInputs" -> BodyPart(inputs.mkString("[", ",", "]"))) ++ options.map("workflowOptions" -> BodyPart(_))
```

Comment:
My question is whether the workflow options struct has changed to support call caching in C24, and whether our case class needs to change in accordance with that.